### PR TITLE
[Feature 1] first feature was implemented

### PR DESCRIPTION
**Pull request by itamittech**
Following changes were made

1. newfeature folder was created
2. New file was created

### CODE REVIEW

File: SecondFIle.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Second file created
```
Comment (itamittechsoma) [inline SecondFIle.txt:1]: Nice progress

File: newfeature.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This file is created in feature 1 branch
```
Comment (itamittechsoma) [inline newfeature.txt:1]: Review started for this

File: SecondFIle.txt (hunk 1 above)
Comment (itamittech) [reply SecondFIle.txt:1]: Thanks

File: newfeature.txt (hunk 2 above)
Comment (itamittechsoma) [reply newfeature.txt:1]: Review finished

Comment (itamittech) [reply newfeature.txt:1]: Thanks closing it now

**Review by itamittechsoma — DISMISSED** [review]
Approved

**Review by itamittechsoma — APPROVED** [review]

**Pull request merged**